srcpkgs/occt/template

```diff
@@ -19,6 +19,16 @@ distfiles="https://git.dev.opencascade.org/gitweb/?p=occt.git;a=snapshot;h=refs/
 checksum=655da7717dac3460a22a6a7ee68860c1da56da2fec9c380d8ac0ac0349d67676
 conflicts="oce>=0"
 
+build_options="tbb"
+desc_option_tbb="Enable support for 3D STEP models"
```

Comment:
desc is misleading  as it's copied from kicad

Reply:
Thanks, updated.